src/GraphQL/DocumentElementType/VideoType.php

```diff
@@ -85,6 +85,7 @@ public static function getInstance(Service $graphQlService, AssetType $assetType
                             'type' => $assetType,
                             'resolve' => static function ($value = null, $args = [], $context = [], ResolveInfo $resolveInfo = null) use ($graphQlService) {
                                 if ($value instanceof Video) {
+                                    /** Image|null $relation */
```

Comment:
```suggestion
                                    /** @var Image|null $relation */
```
Exist the use statement of the Image class?